README.md

```diff
@@ -111,6 +111,29 @@ Prerequisites
    $ sudo pacman -Syu riscv64-elf-gcc riscv32-elf-newlib
    ```
 
+   **newlib-nano**:
```

Comment:
In the section 30 lines earlier that reads "If you would rather compile your own newlib-based libc, follow these steps:", could we add a pointer to this section so that people know to add these flags if they want the size optimized version?